index.d.ts

```diff
@@ -0,0 +1,138 @@
+// Type definitions for looks-same 4.0
+// Project: https://github.com/gemini-testing/looks-same/releases
+// Definitions by: xcatliu <https://github.com/xcatliu>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node"/>
+
+type LooksSameCallback = (error: Error | null, equal: boolean) => any;
+
+/**
+ * The options passed to looksSame function
+ */
+interface LooksSameOptions {
+    /**
+     * By default, it will detect only noticeable differences. If you wish to detect any difference, use strict options.
+     */
+    strict?: boolean;
+    /**
+     * You can also adjust the ΔE value that will be treated as error in non-strict mode.
+     */
+    tolerance?: number;
+    /**
+     * Some devices can have different proportion between physical and logical screen resolutions also known as pixel ratio.
+     * Default value for this proportion is 1.
+     * This param also affects the comparison result, so it can be set manually with pixelRatio option.
+     */
+    pixelRatio?: number;
+    /**
+     * For visual regression tasks it may be useful to ignore text caret in text input elements. You can do it with ignoreCaret option.
+     */
+    ignoreCaret?: boolean;
+    /**
+     * Some images has difference while comparing because of antialiasing.
+     * These diffs will be ignored by default. You can use ignoreAntialiasing option with false value to disable ignoring such diffs.
+     * In that way antialiased pixels will be marked as diffs.
+     */
+    ignoreAntialiasing?: boolean;
+    /**
+     * Sometimes the antialiasing algorithm can work incorrectly due to some features of the browser rendering engine.
+     * Use the option antialiasingTolerance to make the algorithm less strict.
+     * With this option you can specify the minimum difference in brightness (zero by default)
+     * between the darkest/lightest pixel (which is adjacent to the antialiasing pixel) and theirs adjacent pixels.
+     *
+     * We recommend that you don't increase this value above 10. If you need to increase more than 10 then this is definitely not antialiasing.
+     */

```

Comment:
Can you explain please why we should write all this information about each option which is already present in readme?

Reply:
Because with this comments, IDE can show these docs while coding.